packages/views.client-react.unstable/src/utils/initializeParameters.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type {
+  AsyncParameterValueMap,
+  ParameterConfig,
+} from "@osdk/views-client.unstable";
+
+/**
+ * Utility function to initialize a map of parameter values to either a loading or not-started loading state
+ */
+export function initializeParameters<CONFIG extends ParameterConfig>(
```

Comment:
I have a few thoughts about this:
1. If everyone is basically going to have to call this, why isn't it just done for them? Is there some scenario when someone doesn't want this behavior?
2. Is there a world where the backend doesn't provide all the parameters at once? Why have each value require an async check vs the whole thing?

I think really what I want as a consumer is to do this:
`const { isReady, parameters }  = useFoundryViewClient();` and not worry about checking the async of each value?

I'm also unsure that I would want to be responsible for calling `sendReady()`? Like isn't this hook being run all that is required? If I end up building a complex app am I now in the business of keeping track of which of my uses of `useFoundryViewClient()` did or did not sendReady?